packages/map/src/vaadin-map.js

```diff
@@ -99,6 +366,10 @@ class Map extends ElementMixin(ThemableMixin(PolymerElement)) {
     this._configuration = new OpenLayersMap({
       target: this.$.map
     });
+    // Set default controls
+    this._configuration.getControls().clear();
+    this._configuration.getControls().push(new Zoom({ zoomInLabel: '', zoomOutLabel: '' }));
+    this._configuration.getControls().push(new Attribution());
```

Comment:
What's your reasoning for overriding the defaults here? I don't think I have a strong opinion on this. For the Flow component we (hopefully) will add an API to configure these, and maybe have to duplicate some defaults there anyway.

Reply:
The main reasoning is to be able to override the labels/icons in the controls.

Reply:
Let's add a comment with that information then.

Reply:
Done